docs/securing-control-plane.md

```diff
@@ -0,0 +1,155 @@
+# Securing Control Plane
+
+All API communication between Antrea control plane components is encrypted with
+TLS. The TLS certificates that Antrea requires can be automatically generated.
+You can also provide your own certificates. This page explains the certificates
+that Antrea requires and how to configure and rotate them for Antrea.
+
+## Table of Contents
+
+- [What certificates are required by Antrea](#what-certificates-are-required-by-antrea)
+- [How certificates are used by Antrea](#how-certificates-are-used-by-antrea)
+- [Providing your own certificates](#providing-your-own-certificates)
+  - [Using kubectl](#using-kubectl)
+  - [Using cert-manager](#using-cert-manager)
+- [Certificate rotation](#certificate-rotation)
+
+## What certificates are required by Antrea
+
+Currently Antrea only requires a single server certificate for the
+antrea-controller API server endpoint, which is for the following communication:
+- The antrea-agents talks to the antrea-controller for fetching the computed
+ NetworkPolicies
+- The kube-aggregator (i.e. kube-apiserver) talks to the antrea-controller for
+ proxying antctl's requests (when run in "controller" mode)
+
+Antrea doesn't require client certificates for its own components as it
+delegates authentication and authorization to the Kubernetes API, using 
+Kubernetes [ServiceAccount tokens](https://kubernetes.io/docs/reference/access-authn-authz/authentication/#service-account-tokens)
+for client authentication.
+
+## How certificates are used by Antrea
+
+By default, antrea-controller generates a self-signed certificate. You can
+override the behavior by [providing your own certificates](#providing-your-own-certificates).
+Either way, the antrea-controller will distribute the CA certificate as a
+ConfigMap named `antrea-ca` in the `kube-system` Namespace and inject it into
+the APIServices resources created by Antrea in order to allow its clients (i.e.
+antrea-agent, kube-apiserver) to trust itself.
```

Comment:
"itself" is not correct here

suggestion: "to trust itself" -> "to perform authentication"?

Reply:
done, thanks